Sami/Resources/themes/default/macros.twig

```diff
@@ -94,14 +94,17 @@
 
 {% macro breadcrumbs(namespace) %}
     {% set current_ns = '' %}
-    {% for ns in namespace|split('\\') %}
-        {% if current_ns %}
-            {% set current_ns = current_ns ~ '\\' ~ ns %}
-        {% else %}
-            {% set current_ns = ns %}
-        {% endif %}
-        <li><a href="{{ namespace_path(current_ns) }}">{{ ns|raw }}</a></li>
-    {% endfor %}
+    {% spaceless %}
```

Comment:
I would prefer that we use `{%-` or `-%}` where needed as `spaceless` is much slow.